test/fixtures/fetch-taginfo.js

```diff
@@ -0,0 +1,51 @@
+
+/**
+  Utility script to scrape tag information from taginfo
+**/
+
+var taginfo = require('taginfo'),
+    extend = require('extend'),
+    async = require('async');
+
+var q = {
+  query: 'name:',
+  sortname: 'count_all',
+  sortorder: 'desc',
+  page: 1,
+  rp: 500,
+  format: 'json'
+};
+
+function request( page ){
+  return function(done){
+    taginfo.keys.all(
+      done.bind(null, null),
+      extend( false, {}, q, { page: page } )
+    );
+  };
+}
+
+var requests = [
+  request(1),
+  request(2),
+  request(3),
+  request(4),
+  request(5)
```

Comment:
Its requests 5 pages of 500 results, meaning we get all ~2300 of them.

This script is not actually part of the code so I would be happy to delete it, yet it might be useful for someone else at a later date. (also the require'd modules are not in the package.json)